stablehlo/tools/StablehloTranslateMain.cpp

```diff
@@ -83,7 +99,8 @@ llvm::Error evalCustomCallCheckEq(stablehlo::CustomCallOp op,
 
 llvm::Error interpreterFallback(Operation &op, stablehlo::Process *process,
                                 stablehlo::Scope &scope,
-                                llvm::StringRef funcName) {
+                                llvm::StringRef funcName,
+                                llvm::StringMap<int32_t> &instrumentedTensors) {
```

Comment:
Per earlier feedback, please update the name you decide to use for `instrumentedTensors` here as well.

I'll think more on if this is how we should expose this. I don't love adding new input arguments here, but this state needs to be stored somewhere. I think this is good for now, but if there were one more thing added that required state, perhaps we should add an `InterpreterConfig` / `InterpreterState` class to manage stateful `InterpreterDialect` ops.

Reply:
I think maybe we can tackle a more elegant solution as part of #1780?